src/card/card.js

```diff
@@ -1,10 +1,11 @@
 import colorUtils from '../utils/color-utils';
+import encodeUtils from '../utils/encoder';
 
 export function getBackgroundColor(data, cardStyling) {
   let color = cardStyling.backgroundColor;
   if (data.attributes && data.attributes.color) {
     const resolvedColor = colorUtils.resolveExpression(data.attributes.color);
-    color = resolvedColor !== 'none' ? resolvedColor : color;
+    color = resolvedColor !== 'none' ? resolvedColor : encodeUtils.encodeCssColor(color);
```

Comment:
encodeCssColor does not include all options. Instead of changing the encoder returning the result on line 7 would also be okay

Reply:
I thought resolveExpression would be good enough in this case, but I did miss the color set before the if-statement

Reply:
should I just copy the resolveColor function from master?

Reply:
That should work!